server/game/GameObject.js

```diff
@@ -7,6 +7,7 @@ class GameObject {
         this.game = game;
         this.name = name;
         this.id = this.name;
+        this.objectType = '';
```

Comment:
This is incorrect. This is restoring the behaviour that we removed

Reply:
I really think that having the method getType() returning this.type without having defined the type parameter is wrong.

It is not removing the old behavior because Card#get type is overriden

Reply:
Rolled back this change